monocle/bounds.py

```diff
@@ -38,12 +38,21 @@ def __init__(self, polygon=conf.BOUNDARIES):
         self.south, self.west, self.north, self.east = polygon.bounds
         self.center = polygon.centroid.coords[0]
         self.multi = False
+        self.prepared = False
+        try:
+            from shapely.prepared import prep
+            self.prepared_boundaries = prep(polygon)
+            self.prepared = True
+        except:
+            pass
```

Comment:
If we're keeping the `try`/`except` and using the name changes I suggested above, replace the `pass` with:
```python
self.boundaries = polygon
```